src/hotspot/share/runtime/abstract_vm_version.cpp

```diff
@@ -128,11 +129,15 @@ const char* Abstract_VM_Version::vm_name() {
 #endif
 
 static const char vm_vendor_string[sizeof(VENDOR) < VENDOR_PADDING ? VENDOR_PADDING : sizeof(VENDOR)] = VENDOR;
+const char* Abstract_VM_Version::_vm_vendor = vm_vendor_string;
```

Comment:
As the system property is no longer set by VM, it can be initialized to null and this field should only be read after initPhase1.  It would be much simplified.